binlogctl/meta_test.go

```diff
@@ -48,3 +50,42 @@ func (s *metaSuite) TestSaveMeta(c *C) {
 	// the output depends on the local's timezone
 	c.Assert(lines[2], Matches, "1970-01-0.*")
 }
+
+type dummyCli struct {
+	pd.Client
+	physical, logical int64
+	err               error
+}
+
+func (c dummyCli) GetTS(ctx context.Context) (int64, int64, error) {
+	return c.physical, c.logical, c.err
+}
+
+func NewFakePDClient([]string, pd.SecurityOption) (pd.Client, error) {
```

Comment:
This doesn't have to be a public function?

Reply:
done